climateconnect/climateconnect
# Feature: Adapt Project Page To Event Calendar

## What and Why

This PR makes the necessary adjustments to the project page to account for different types of projects.

## Detailled changes
- Store project types in a constant instead of the database as suggested by @ddhanesha 
- Adapt terms in static texts to project type, e.g. "Event Description" instead of "Project Description" on events
- Correctly display detailled location including additional info
- Display start and end date of events in the summary section -> display date differently depending on language
- Display project type in the summary section
- Don't show project status anymore
- Don't show project dates for events anymore
- Made admin buttons into icon buttons on mobile
- Polished parent organization display
- Ran 'yarn format'
- Cleaned up code by typing some functions, isolating some components and cutting down on passed through arguments

## Testing Guide
Create one project of each project type and see if they are displayed correctly. Also test this when logged out.

## Screenshots
![Screenshot from 2023-08-17 11-31-05](https://github.com/climateconnect/climateconnect/assets/10531079/487df7b1-3ea5-48db-b031-8cdf278a62aa)
![Screenshot from 2023-08-17 11-37-15](https://github.com/climateconnect/climateconnect/assets/10531079/df98b2d5-4b83-41dc-aadb-8ac0f54ec1d0)
